src/Sulu/Bundle/WebsiteBundle/Resources/config/services.xml

```diff
@@ -122,6 +123,7 @@
             <argument type="service" id="sulu.content.mapper"/>
             <argument type="service" id="sulu_website.navigation_mapper"/>
             <argument type="service" id="sulu_core.webspace.request_analyzer" on-invalid="ignore"/>
+            <argument type="service" id="security.token_storage" on-invalid="ignore"/>
```

Comment:
I personally would use `ignore` only if we use setter Injection. Find it a little bit confusing when used with constructor injection, because it sounds for me like the whole service  definition is then ignored, but it behaves in constructor the same as `on-invalid="null",  so I would go with `on-invalid="null"` in constructor injection. But we can also keep it this way, just did confuse me at the first if what the difference is here.